apps/reaper/lib/reaper/event/event_handler.ex

```diff
@@ -32,15 +39,21 @@ defmodule Reaper.Event.EventHandler do
       :discard
   end
 
-  def handle_event(%Brook.Event{type: ingestion_delete(), data: %SmartCity.Ingestion{} = ingestion}) do
+  def handle_event(%Brook.Event{
+        type: ingestion_delete(),
+        data: %SmartCity.Ingestion{} = ingestion
+      }) do
     ingestion_delete()
     |> add_event_count(ingestion.id)
 
     Reaper.Event.Handlers.IngestionDelete.handle(ingestion)
     Extractions.delete_ingestion(ingestion.id)
   end
 
-  def handle_event(%Brook.Event{type: data_extract_start(), data: %SmartCity.Ingestion{} = ingestion}) do
+  def handle_event(%Brook.Event{
+        type: data_extract_start(),
+        data: %SmartCity.Ingestion{} = ingestion
+      }) do
```

Comment:
Is my lint too excitable? I'm not sure if I'm using the wrong linter and that's why there are some changed.